src/database/connection.py

```
@@ -36,7 +36,9 @@ class PostgresConnection:
     def __init__(self, minconn: int, maxconn: int, **conn_kwargs: Any):
         # Validate required credentials early
         if not conn_kwargs.get("password"):
-            raise ValueError("DB_PASSWORD environment variable is required for database connections")
+            raise ValueError(
+                "DB_PASSWORD environment variable is required for database connections"
+            )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)